src/util/stream-to-promise.js

```diff
@@ -0,0 +1,9 @@
+/* @flow */
+
+export async function streamToPromise(
```

Comment:
Why did you have to replace [stream-to-promise](https://github.com/eight04/web-ext/blob/e58888bae779b65985ca5951e003d3b420506fce/package.json#L66)? Is it broken on Windows? Can you [patch it upstream](https://github.com/bendrucker/stream-to-promise) instead? If you really need to replace it (I hope not) then the patch needs some changes:
* All the various error callbacks need to result in promise rejections
* I'd like to see some debug logging
* We'll need unit test cases for all possible stream to promise scenarios
* `stream-to-promise` should be removed from `package.json`